modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -239,6 +239,11 @@
 			<return type="bool" />
 			<argument index="0" name="type" type="StringName" />
 			<description>
+				Returns whether the given class exists in [ClassDB].
+				[codeblock]
+				type_exists("Sprite2D") # Returns true
+				type_exists("Variant") # Returns false
```

Comment:
I feel that's a bit of a weird example as Variant is a valid class - it may not be exposed as is in ClassDB but I'm not sure that's the best place to communicate that, if any.

Reply:
Should I replace it with something more obvious such as `NonExistentClass`?

Reply:
Yeah I think thats better.
Might be worth adding an example with a primitive type like `int` too.

Reply:
That would also return false like the `Variant` example, isn't it confusing too?

Reply:
Yes. Then I would suggest clarifying this in the description itself.